app/models/volunteer.rb

```diff
@@ -55,14 +60,14 @@ def active_for_authentication?
   end
 
   def inactive_message
-    if not assigned
+    if !assigned?
```

Comment:
I actually originally inverted it myself, but ended up keeping it as is to match the example at https://github.com/plataformatec/devise/wiki/How-To:-Require-admin-to-activate-account-before-sign_in, referenced in the comment on line 57.